content/includes/header.html

```diff
@@ -34,6 +34,9 @@
 <meta content="{{ tag }}" property="article:tag">
 {% endfor %}
 {% endif %}
+{% if includes_wizard %}
```

Comment:
I hate to be so verbose, but just in case we need to make other wizards, should we be more specific and call this something like `includes_edu_wizard`?